apps/sertra/components/ModCard.tsx

```diff
@@ -17,7 +17,11 @@ export const ModCard: React.FC<ModCard> = ({
 }) => (
   <div className={styles.mod}>
     <div className={styles.column}>
-      <img className={styles.imagePlaceholder} src={icon_url}></img>
+      {icon_url ? (
+        <img className={styles.imageIcon} src={icon_url}></img>
+      ) : (
+        <img className={styles.imageIcon} src="/favicon.ico"></img>
```

Comment:
Isn't that `favicon.ico` the Vercel's default icon? Or has that been replaced already. Anyhow I'd prefer `public/ts-logo.svg` as a quick solution. Proper solution is probably asking grav whether we want to use the same approach as the new upcoming UI, or have some other placeholder for sertra.

Also since the only thing changing seems to be the `src`, the ternary operator is unnecessarily verbose. This would be more readable IMO:
`<img className={styles.imageIcon} src={icon_url ?? "/favicon.ico"} />`

Also also I think you should fix the things ESLint is complaining about?

Reply:
Ah I didn't know I could do that one line ternary thing 👍 Also switched to using the nexts Image component. I'm not sure what are the benefits of that, but at least they had an example on nextjs.com.